eng/pipelines/ci-template.yml

```diff
@@ -99,28 +95,26 @@ steps:
     - script: black $(Build.SourcesDirectory)/autorest.python
       displayName: Black
       workingDirectory: $(Build.SourcesDirectory)/autorest.python/packages/${{parameters.folderName}}
-      condition: and(succeeded(), ${{ parameters.pythonCodeChecks }})
+      condition: and(succeeded(), ${{ parameters.pythonCodeBlackCheck }})
```

Comment:
i want to work to where instead of calling `pyright` or `pylint` here, we have `lint` and `format` scripts in `package.json` which call. I'm going to work on this tomorrow

Reply:
I see. And please feel free to commit to this PR directly.